spec/lib/evss/disability_compensation_form/form_8940_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'evss/disability_compensation_form/form8940'
+
+describe EVSS::DisabilityCompensationForm::Form8940 do
+  let(:form_content) do
+    JSON.parse(
+      File.read('spec/support/disability_compensation_form/front_end_submission_with_8940.json')
+    )
+  end
+  let(:expected_output) { JSON.parse(File.read('spec/support/disability_compensation_form/form_8940.json')) }
+  let(:user) { build(:disabilities_compensation_user) }
+
+  before do
+    User.create(user)
```

Comment:
how about changing `build(:disabilities_compensation_user)` to `create(:disabilities_compensation_user)` and deleting this line?

Reply:
Sounds good